sdk/servicebus/azure-servicebus/azure/servicebus/control_client/models.py

```diff
@@ -49,8 +53,17 @@ def __init__(self, lock_duration=None, max_size_in_megabytes=None,
             duplicate_detection_history_time_window
         self.max_delivery_count = max_delivery_count
         self.enable_batched_operations = enable_batched_operations
-        self.size_in_bytes = size_in_bytes
-        self.message_count = message_count
+        self.size_in_bytes = None
+        self.message_count = None
+        self.authorization_rules = []
+        self.auto_delete_on_idle = auto_delete_on_idle
+        self.availability_status = None
+        self.enable_partitioning = enable_partitioning
+        self.forward_dead_lettered_message_to = forward_dead_lettered_message_to
+        self.forward_to = forward_to
+        self.path = path
+        self.status = status
+        self.user_metadata = user_metadata
```

Comment:
If would be great if we could update the [control plane tests](https://github.com/KieranBrantnerMagee/azure-sdk-for-python/tree/kibrantn/partitioned-model-support/sdk/servicebus/azure-servicebus/tests/control_plane_tests) to verify the assignment/type/serialization of these new properties - as all the data should already be in the updated recordings.